.yarnrc.yml

```diff
@@ -6,7 +6,7 @@ enableHardenedMode: true
 
 enableMirror: true
 
-enableOfflineMode: true
+enableOfflineMode: false
```

Comment:
This feature caused me all kinds of problems when trying to get this working, so I don't think it's worth it. A real footgun here.